example/build.gradle

```diff
@@ -3,12 +3,12 @@ apply plugin: "kotlin-android"
 apply plugin: "kotlin-android-extensions"
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    compileSdkVersion "android-S" // TODO change this to 31 when Android 12 leaves beta
+    // buildToolsVersion "29.0.2"
     defaultConfig {
         applicationId "io.radar.example"
         minSdkVersion 16
-        targetSdkVersion 29
+        targetSdkVersion 30
```

Comment:
I think this is supposed to be 'S', not 30?

https://developer.android.com/about/versions/12/setup-sdk#groovy

Reply:
<img width="316" alt="image" src="https://user-images.githubusercontent.com/739611/122811830-f0b52080-d29e-11eb-8e31-194ef3ab6506.png">